# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/wasm/jsapi/constructor/multi-value.any.js.ini
# Commit: 16fd9844f2d2
# Full Hash: 16fd9844f2d2029feaba3c864b1a0f830f9da235
# Author: Andy Wingo <wingo@igalia.com>
# Date: 2020-03-18 16:05:52
# Regressor Bug: 1620197
# File Overlap Count: 1
# Description:
#   Bug 1620197 - Enable multiple results from WebAssembly functions r=lth
#   
#   This patch enables multi-value calls and returns, adding some tests, and
#   conditionally disabling a couple expect-fail reftests that now pass.
#   
# ==============================================================================

diff -r 0d19968c99a7 -r 16fd9844f2d2 testing/web-platform/meta/wasm/jsapi/constructor/multi-value.any.js.ini
--- a/testing/web-platform/meta/wasm/jsapi/constructor/multi-value.any.js.ini	Wed Mar 18 13:39:50 2020 +0200
+++ b/testing/web-platform/meta/wasm/jsapi/constructor/multi-value.any.js.ini	Wed Mar 18 10:29:53 2020 +0000
@@ -14,7 +14,8 @@
     expected: FAIL
 
   [multiple return values inside wasm]
-    expected: FAIL
+    expected:
+      if not nightly_build: FAIL
 
   [multiple return values from js to wasm]
     expected: FAIL
